### ISSUE TITLE
Cassandra: illegal reflective access operation

### ISSUE THREAD

**Issue by JuliaLawall** [dacapobench/dacapobench#191]
```
Using scaled threading model. 128 processors detected, 128 threads used to drive the workload, in a possible range of [1,unlimited]
Cassandra starting...
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/tmp/./scratch/jar/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/tmp/./scratch/jar/slf4j-simple-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.cassandra.utils.FBUtilities (file:/tmp/./scratch/jar/cassandra-3.11.3.jar) to field java.io.FileDescriptor.fd
WARNING: Please consider reporting this to the maintainers of org.apache.cassandra.utils.FBUtilities
WARNING: Use --illegal-access=warn to enable warnings of further illegal openjdk 11.0.11 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-post-Debian-1)
OpenJDK 64-Bit Server VM (build 11.0.11+9-post-Debian-1, mixed mode, sharing)
reflective access operations
WARNING: All illegal access operations will be denied in a future release
YCSB starting...
```
Is this a problem?  I am using Debian 11.

```
openjdk 11.0.11 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-post-Debian-1)
OpenJDK 64-Bit Server VM (build 11.0.11+9-post-Debian-1, mixed mode, sharing)
```

**Comment by steveblackburn** [comment]
Nope.   Sadly that's a [known issue](https://stackoverflow.com/questions/50251798/what-is-an-illegal-reflective-access) with the upstream, cassandra.

**Comment by caizixian** [comment]
The current workaround can be found here https://github.com/apache/cassandra/blob/0c444a75e79da8c157813a72d61f2b2f86e187ba/src/java/org/apache/cassandra/utils/FBUtilities.java#L1076

**Comment by steveblackburn** [comment]
Fixed in 3cf1f53
